changelog/unreleased/breaking-changes/1703--flexible-plugin-loading.md

```diff
@@ -0,0 +1 @@
+VAST no longer loads static plugins by default.
```

Comment:
Can you point me to the corresponding code change that this refers to?

Reply:
https://github.com/tenzir/vast/pull/1703/files#diff-52e07dd1a5a1de790b1d57e81d445a22a0b8bb6791a5f13f44ca61b8ae1ceb2fL85-L86 vs. https://github.com/tenzir/vast/pull/1703/files#diff-4599665b12bc79bea311152e2c968a74ce853bc06ac316c18928272485017251R215-R216